src/resources/Incorporations/unlimitedCompany.ts

```diff
@@ -53,36 +53,23 @@ export const UnlimitedCompanyResource: IncorporationResourceIF = {
   incorporationAgreement: {
     helpSection: [
       {
-        header: 'What is the sample Incorporation Agreement and Table 1 Articles?',
+        header: 'What is the sample Incorporation Agreement and Unlimited Liability Company Articles?',
         helpText: [
-          `The sample Incorporation Agreement and Table 1 Articles is a template that you can use to create an
-            incorporation agreement and articles for your company. It uses all the standard provisions suggested by
-            legislation.`,
-          `If you would like to customize any provisions in the Articles, you cannot use this sample. We recommend
-            seeking professional assistance from a lawyer or accountant to help you prepare your Articles.`
+          `The sample Incorporation Agreement and Company Articles is a template that you can use to create an 
+            incorporation agreement and articles for your company. It uses all the standard provisions by legislation. 
+            There are three types of samples depending on if you’re incorporating a Limited Company, a Benefit Company, 
+            or a Community Contribution Company. There is currently no Incorporation Sample for Unlimited 
+            Liability Companies.`,
```

Comment:
I can see this is the same text for all 3 entity types, but it seems wrong to me that a ULC IA refers to a BC LTD or BEN or CCC ...?

Please confirm with UI team that this is really what they and the business wants.

Reply:
Confirming with Ethan/ui team

Reply:
Any update on this?

Reply:
Re-requested your review.

Reply:
I still see references to "three types of samples ... Benefit Company" in the CCC, BC and ULC resources. Is this really what the business wants?

Reply:
I had asked Ethan to check the content for all 3 types and the only update from him was for BC LTD company which is removed now. I can confirm with him again and change if he has any updates.

Reply:
Or you could merge this change as-is and let the UX Assurance step catch any issues. It's probably the same amount of work for you either way, so choose whichever option you find less stressful :)